### ISSUE TITLE
SpikiLi: A Spiking Simulation of LiDAR based Real-time Object Detection for Autonomous Driving. (arXiv:2206.02876v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#19752]
SpikiLi: A Spiking Simulation of LiDAR based Real-time Object Detection for Autonomous Driving. (arXiv:2206.02876v1 [cs.CV])<br>
https://ift.tt/FrBmaG8<br>
<p>Spiking Neural Networks are a recent and new neural network design approach that promises tremendous improvements in power efficiency, computation efficiency, and processing latency. They do so by using asynchronous spike-based data flow, event-based signal generation, processing, and modifying the neuron model to resemble biological neurons closely. While some initial works have shown significant initial evidence of applicability to common deep learning tasks, their applications in complex real-world tasks has been relatively low. In this work, we first illustrate the applicability of spiking neural networks to a complex deep learning task namely Lidar based 3D object detection for automated driving. Secondly, we make a step-by-step demonstration of simulating spiking behavior using a pre-trained convolutional neural network. We closely model essential aspects of spiking neural networks in simulation and achieve equivalent run-time and accuracy on a GPU. When the model is realized on a neuromorphic hardware, we expect to have significantly improved power efficiency.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
